source/configuration_manual/dovecot_ssl_configuration.rst

```diff
@@ -62,11 +62,11 @@ There are a couple of different ways to specify when SSL/TLS is required:
 
   .. NOTE:: With both :dovecot_core:ref:`ssl=yes <ssl>` and :dovecot_core:ref:`ssl=required <ssl>` it's still possible that the client attempts to do a cleartext authentication before enabling SSL/TLS, which exposes the cleartext password to the internet.
 
-             Dovecot attempts to indicate this to the IMAP clients via the LOGINDISABLED capability, but many clients still ignore it and send the password anyway. There is unfortunately no way for Dovecot to prevent this behavior. The POP3 standard doesn't have an equivalent capability at all, so the POP3 clients can't even know if the server would accept a cleartext authentication.
+      Dovecot attempts to indicate this to the IMAP clients via the LOGINDISABLED capability, but many clients still ignore it and send the password anyway. There is unfortunately no way for Dovecot to prevent this behavior. The POP3 standard doesn't have an equivalent capability at all, so the POP3 clients can't even know if the server would accept a cleartext authentication.
```

Comment:
This text belongs to the note above, so it shouldn't be unindented. Might belongs to the same paragraph also..